src/content/translate.ts

```diff
@@ -16,7 +16,17 @@ export interface TranslateResult {
 }
 
 export async function translate(query: TranslateQuery): Promise<TranslateResult> {
-    const apiKey = await utils.getApiKey()
+    const apiKeys = await utils.getApiKey(); 
+    const keys = apiKeys.split(';'); // Split the API keys into an array
+    const key = keys[Math.floor(Math.random() * keys.length)]; // Select a random API key from the array
```

Comment:
Can you put the logic for selecting apiKey in the `utils.getApiKey` function?

Reply:
Of course, I can. The code has already been submitted.